docs/configure/nodes.md

```diff
@@ -1,13 +1,13 @@
 <!--
-title: "Configure your nodes"
+title: "Configure the Netdata Agent"
 description: "Netdata is zero-configuration for most users, but complex infrastructures may require you to tweak some of the Agent's granular settings."
 custom_edit_url: https://github.com/netdata/netdata/edit/master/docs/configure/nodes.md
 -->
 
-# Configure your nodes
+# Configure the Netdata Agent
 
 Netdata's zero-configuration collection, storage, and visualization features work for many users, infrastructures, and
-use cases, but there are some situations where you might want to configure your nodes.
+use cases, but there are some situations where you might want to configure the Netdata Agent running on your node(s).
```

Comment:
```suggestion
use cases, but there are some situations where you might want to configure the Netdata Agent running on your machine(s).
```

Reply:
"Machines" is too precise, since a node can be a container, too.